### ISSUE TITLE
question about input of CoarseGenerator and FineGenerator

### ISSUE THREAD

**Issue by zhengbowei** [daa233/generative-inpainting-pytorch#36]
Dear author:
I have a little question for you：
In the CoarseGenerator, the network input is:
x = self.conv1(torch.cat([x, ones, mask], dim=1))

In the FineGenerator, the network input is:
xnow = torch.cat([x1_inpaint, ones, mask], dim=1)

My question is that they torch.cat "mask": 0 indicates the outside known region and 1 indicates the inside missing region. 
Didn't it mean that this operation masks the known region and shows the missing region? Is it right？

Thank you for your answer。

**Comment by daa233** [comment]
No, I don't think so. Maybe `x * mask` would "masks the known region and shows the missing region", in my view, concat will reserve all the information.
